emsdk/patches/posixish_memfs.patch

```diff
@@ -0,0 +1,37 @@
+Modify MEMFS to update timestamp of parent directory on file creation and removal.
+More POSIXish compliant for a one timestamp fs, behaves like a modification date.
+
+--- a/emsdk/fastcomp/emscripten/src/library_memfs.js    2019-08-19 18:11:40.000000000 +0200
++++ b/emsdk/fastcomp/emscripten/src/library_memfs.js	2020-12-18 14:07:14.000000000 +0100
+@@ -88,6 +88,7 @@
+       // add the new node to the parent
+       if (parent) {
+         parent.contents[name] = node;
++        parent.timestamp = node.timestamp;
+       }
+       return node;
+     },
+@@ -213,12 +214,15 @@
+         }
+         // do the internal rewiring
+         delete old_node.parent.contents[old_node.name];
++        old_node.parent.timestamp = Date.now()
+         old_node.name = new_name;
+         new_dir.contents[new_name] = old_node;
++        new_dir.timestamp = old_node.parent.timestamp;
```

Comment:
I think it would be clearer if this said `new_dir.timestamp = Date.now()` instead.

Reply:
I think it is better if both the old and the new dir have the same timestamp after rename.
Would you prefer if I use a new variable now = Date.now() and use it to update both old_node.parent and new_dir timestamps?

Reply:
That would work too